cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -33,20 +36,23 @@ const (
 	// the '--timeout' flag. The value should be bigger than the check timeout
 	// defined in the manifest, to ensure the check will have a 'TIMEOUT'
 	// status when the execution takes longer than expected.
-	trivyTimeout = "2h"
+	trivyTimeout     = "2h"
+	maxNonBinaryFile = 1024 * 1024 * 20
```

Comment:
I think it would be helpful to add a comment with a "human readable" size next to the value/operation.